h/accounts/schemas.py

```diff
@@ -85,6 +86,13 @@ def unblacklisted_username(node, value, blacklist=None):
         raise colander.Invalid(node, msg)
 
 
+def privacy_acceptance_validator(node, value):
+    '''Colander validator that ensures privacy acceptance checkbox checked'''
```

Comment:
I noticed a minor inconsistency that we are using double-quotes for some docstrings and single-quotes for others. It's trivial but we ought to get a linter to save having to think about it.